python_modules/dagster/dagster/_core/definitions/asset_daemon_context.py

```diff
@@ -93,7 +93,7 @@ def __init__(
         cursor: AssetDaemonCursor,
         materialize_run_tags: Optional[Mapping[str, str]],
         observe_run_tags: Optional[Mapping[str, str]],
-        auto_observe: bool,
+        auto_observe_asset_keys: Optional[AbstractSet[AssetKey]],
```

Comment:
Should we just pass them both through `target_asset_keys`?

Reply:
If not, then maybe we should have `auto_observe_asset_keys` and `auto_materialize_asset_keys`?